app/controllers/concerns/workflow_execution_actions.rb

```diff
@@ -98,4 +109,33 @@ def set_default_sort
   def redirect_path
     raise NotImplementedError
   end
+
+  def format_samplesheet_params
+    @samplesheet_headers = @workflow_execution.samples_workflow_executions.first.samplesheet_params.keys
```

Comment:
This line needs safe access, if the workflow execution has 0 samples this throws a undefined method exception.